docker/docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+
+networks:
+  rolelive-net:
+    driver: bridge
+
+services:
+
+# Python
+  python:
+    image: 'oriolfilter/python-rolelive:first_try'
```

Comment:
Image should be built from the Dockerfile you created using ``build``, not downloaded from Dockerhub.